src/modules/add_remove.js

```diff
@@ -18,20 +18,11 @@ export default class Tasks {
 
   remove() {
     if (!this.storage) {
-      const arr = [];
       let allTasks = localStorage.getItem('tasks');
       allTasks = JSON.parse(allTasks);
-      const selected = document.querySelectorAll('.cross_out');
-      selected.forEach((element) => {
-        const index = parseInt(element.id, 10);
-        arr.push(index);
-      });
-      for (let i = arr.length - 1; i >= 0; i -= 1) {
-        const index = arr[i];
-        allTasks.splice(index, 1);
-      }
-      allTasks = JSON.stringify(allTasks);
-      localStorage.setItem('tasks', allTasks);
+      let filtr = allTasks.filter((done) => done.complete === false);
+      filtr = JSON.stringify(filtr);
+      localStorage.setItem('tasks', filtr);
```

Comment:
- [ ] I can see that you've implemented the clear-all button well and that it functions as intended, but I also noticed that it can be used to clear one task that has already been marked. I think this compromises the user experience of your application, so please fix it so that your project looks captivating.  :wink:  :smile: 

I mean Make the three dots in your design below functional such that when they are clicked, they will display the delete button, which will allow us to remove just one task. :wink: 
![Screenshot from 2022-12-08 01-35-13](https://user-images.githubusercontent.com/88894525/206320487-6e40d4eb-38b4-405c-bc48-d5f667141556.png)

Reply:
@Alfredbis29 Does this mean that if I have a task checked, the button should not delete the task? Could you specify a little more about what you require it to do, please?

Reply:
@sambeck87  Implement a button that deletes only one completed task, as we can do with the clear-all button, which does not truly describe that capability; the clear-all button should remove all completed tasks, while the delete button should only delete one completed task. I think this is now more explicit . :wink:

Reply:
@Alfredbis29 sorry but. If you see the video the button can delete a task even if this is not complete, you can delete all the completed tasks by clicking on clear all completed
[Grabación de pantalla desde 07-12-22 18:20:01.webm](https://user-images.githubusercontent.com/113067895/206325667-ebc9d3ff-67fc-4ef5-9e18-68345859b797.webm)
you can see the whole video here:
https://www.youtube.com/watch?v=AcUd-_Yjjqg
I think my project works as the video except for the drag function. In my opinion, your suggestion could be optional.
Kindly I suggest check the video, and if you see something different let me know.
If after watching the video you consider your suggestion as mandatory, I'll make the change

Reply:
@sambeck87 can see in the video there is a delete icon that appears once clicked on those 3 dots then that one delete one task even if it's not completed, this is what I suggest, I know this is very easy for you to implement because you have already the logic of doing that. :wink:

Reply:
Yeah @sambeck87  let also  implement that button which can delete only one task even if is not completed, if you do that when you are ready request another review :wink: , have you understand atleast what to do ?

Reply:
@Alfredbis29 excuse my stubbornness, but the delete button does not appear when you click on the three dots, it appears when you click on the task area.

Reply:
[Grabación de pantalla desde 07-12-22 18:49:24.webm](https://user-images.githubusercontent.com/113067895/206329233-8e8b53fc-2c8a-47c4-a755-e901f233d4bd.webm)
this is my project. I have the delete button to delete one task, just have to click on the task area

Reply:
@sambeck87  You have done it , that I suggested ,kudos , now you ask another review :100:

Reply:
@Alfredbis29 No man, it was the same project... But I did the changes. Thanks